README.md

```diff
@@ -1 +1,39 @@
-# ehj1-uMnieDziala
\ No newline at end of file
+
+# **Employee Leave Management App**
+
+## Table of Contents 🕮
+- [About](#About)
+- [Technologies](#Technologies)
+- [Setup](#Setup)
+- [Status](#Status)
+- [Authors](#Authors)
+
+## About 🏖️
+Project presents an application for managing employee leave.
+Main assumptions of the project:
+- menu on console
+- loading employee data from a file 
+- loading Json file with holidays
+- adding/editing employee from base
+- manually editing data about imported holidays
+- leave planning for the employee
+- loading employee calendar (in iCalendar format) 
+
+## Technologies 💻
+Project is created with:
+- Java version 17
+- Maven
+- log4j
+- assertj
+- Junit
+
+## Setup ⚙️
+To run this project clone GitHub repository _ehj1-uNasDziala_ and run it in IDE of your choice.
```

Comment:
przydałoby się jeszcze info bardziej użytkowe. Coś w stylu:

- Zainstaluj JDK w wersji 17 lub wyższej (link do oficjalnego pobrania)
- pobierz zbudowaną aplikację: (link do najnowszej wersji w repo) lub zbuduj są samodzielnie (link do instrukcji gdzie jest info jak zbudować)
- uruchom plik `java -jar vacations.jar`
- w folderze x powinien znajdować się plik z informacjami nt. dni wolnych. Przykładowy format tutaj (link do repo z domyślnymi świętami). Jeśli nie będzie takiego pliku, to zostanie załadowany plik domyślny.
- do myślnie system szuka pliku o nazwie `aaa`, ale jeśli chcesz podać plik o innej nazwie, to podaj tą nazwę w taki oto sposób...

Reply:
ewentualnie ten opis może się rozszerzać w miarę dodawania kolenych fukcjonalności, a teraz tyko to, co już jest